AppKit/CPTableView.j

```diff
@@ -6400,18 +6400,12 @@ var CPTableViewDataSourceKey                = @"CPTableViewDataSourceKey",
 
 - (BOOL)setThemeState:(ThemeState)aState
 {
-    if (aState.isa && [aState isKindOfClass:CPArray])
```

Comment:
I believe we should keep "that", or change it with `setThemeStates:` when we give an array (and put a CPLog.warn(deprecated - don't use it like this).

These new features are going to break few stuff for sure, we should prevent that.

Same remark for `unsetThemeState:` and `setValue:forThemeAttribute:inState:` and `hasThemeState`